routes/teams.js

```diff
@@ -35,7 +35,7 @@ router.get('/quiz', function(req, res) {
 
   if (req.query.trending){
     if (type){ // leaders or goats
-      db.collection('teams').findOne( { _id : rb_team_id}, function (err, item){
+      db.collection('teams').findOne( { team_id : rb_team_id}, function (err, item){
```

Comment:
from my end it works on league-search page but is slow. I'll look into that later today.

I had to change it to use team_id because of the mongo upsert. Mongo doesn't allow the _id field to be changed during an upsert. 

Opt. A) I'd either have to split my script to go find all the ones in mongo that exist and upsert them without changing the _id... then go find all the ones that don't exist and insert them .. that makes 4 different cases I'd have to deal with with potentially missing item from either teams collection and/or/nor leaders collection........ 

Opt B) it sounded easier to just change everything to use the team_id and ignore the _id for all intents and purposes.